### DISCUSSION TITLE
Automatically translate the page as we progress through the reading

### DISCUSSION THREAD

**Discussion by bxb2world** [johnfactotum/foliate#979] [Ideas]
Would it be possible to configure (or add it in future versions) the functionality to automatically translate the page as we progress through the reading?

**Comment by johnfactotum** [comment]
I don't think it's possible with the current API we're using. The translation service needs to be HTML aware.

For now I would suggest using an in-browser reader (e.g. https://johnfactotum.github.io/foliate-js/reader.html) together with a translation extension such as https://addons.mozilla.org/en-US/firefox/addon/firefox-translations/.
